### PR TITLE
Add section: About This Handbook, under construction

### PR DESCRIPTION
**Pull request by patcon**
Re-ticketed from: https://github.com/hyphacoop/hypha.coop/pull/7

Can't find her comment now, but @dcwalk raised point that if we link it from our website, we should make it clear who the primary audience is.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
 
 Hypha Organizational Handbook (H₂O) describes the vision, processes, and culture of Hypha Worker Co-operative. 🌿🍄
 
+### About This Handbook
+<!-- Derived from: https://handbook.enspiral.com/#about-this-handbook -->
+
+This Handbook's primary audience is Hypha worker-members and their
```
Comment (dcwalk) [inline README.md]:
```suggestion
This Handbook's primary audience is Hypha member-owners and their
```

Comment (dcwalk) [reply README.md]: We've approved language which calls ourselves "member-owners" at a previous meeitng

File: README.md [hunk 2]
```diff
@@ -2,6 +2,20 @@
...
+outside collaborators, but it will be publicly available for others who
```
Comment (dcwalk) [inline README.md]:
```suggestion
collaborators, but it will be publicly available for others who
```

File: README.md [hunk 3]
```diff
@@ -2,6 +2,20 @@
...
+might find it useful. The goal is to give all of us as much clarity and
```
Comment (dcwalk) [inline README.md]:
```suggestion
might find it useful. The goal is to provide as much clarity and
```

File: README.md [hunk 4]
```diff
@@ -2,6 +2,20 @@
...
+context as possible, while sharing our structures and practices with the
+outside world as well.
+
+![Old-timey web 1.0 UNDER CONSTRUCTION banner](images/under-construction.gif)<br />
+This resource is in perpetual beta, and is constantly changing. If you
```
Comment (dcwalk) [inline README.md]:
```suggestion
This resource is in perpetual beta and constantly changing. If you
```

File: README.md [hunk 5]
```diff
@@ -2,6 +2,20 @@
...
+see something that could be improved: that's your invitation to improve
```
Comment (dcwalk) [inline README.md]:
```suggestion
see something that could be improved, consider this your invitation to improve
```

File: README.md (hunk 1 above)
Comment (patcon) [reply README.md]: I'm easy, but tried to search our notes, and only found this, where the notes didn't reflect that _per se_: https://github.com/hyphacoop/organizing/blob/master/2019-09-04-all-hands-meeting.md

Something else you were thinking? Or were notes missing?

File: README.md (hunk 2 above)
Comment (patcon) [reply README.md]: okie doke. added here: https://github.com/hyphacoop/handbook/issues/4#issuecomment-556360041

File: README.md (hunk 1 above)
Comment (dcwalk) [reply README.md]: Pretty sure we discussed this when we finalized membership classes (https://github.com/hyphacoop/organizing/blob/master/2019-05-27-all-hands-meeting.md), but I don't see it in notes. We updated the business plan to "member-owner" afterwards and issues use that language too....

Comment (patcon) [reply README.md]: don't recall deciding such a long time ago, thought it was much more recent, but glad to have one or the other decided :) let's get this in glossary in near future!

**Pull request merged**